Alicerce/Sources/Protocols/View.swift

```diff
@@ -1,12 +1,12 @@
 //
-//  ViewCellProtocol.swift
+//  View.swift
 //  Alicerce
 //
 //  Created by Luís Afonso on 16/12/2016.
 //  Copyright © 2016 Mindera. All rights reserved.
 //
 
-public protocol ViewCellProtocol: ViewCellReuseIdentifier {
+public protocol View: ReusableView {
```

Comment:
Yeap, this protocol is meant to be used by any view, to improve init when we want to do it on code.
The usual use case is `let someWeirdView = UIView() and it works`